config.ru

```diff
@@ -6,14 +6,14 @@
 
 require ::File.expand_path('../config/environment',  __FILE__)
 require ::File.expand_path('../lib/unicorn_killer',  __FILE__)
-require ::File.expand_path('../lib/rack/chrome_frame',  __FILE__)
+require ::File.expand_path('../lib/rack/internet_explorer_version', __FILE__)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.